code/string-calculator/infrastructure/src/main/java/ru/unn/agile/calc/infrastructure/TxtLogger.java

```diff
@@ -0,0 +1,64 @@
+package ru.unn.agile.calc.infrastructure;
+
+import ru.unn.agile.calc.viewmodel.ILogger;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.List;
+
+public class TxtLogger implements ILogger {
+    private static final String DATE_TIME_FORMAT = "yyyy-MM-dd HH:mm:ss";
```

Comment:
"yyyy-MM-dd HH:mm:ss.SSS" should be more usefull

Reply:
done